### DISCUSSION TITLE
why my performance tab no web vitals panel?

### DISCUSSION THREAD

**Discussion by Rainnut** [getsentry/support#22] [Support]
like this:
<img width="1717" alt="image" src="https://user-images.githubusercontent.com/12372532/174259737-af14be7e-86d5-4b80-b032-2662085e8602.png">

no web vitals and Frontend?
![image](https://user-images.githubusercontent.com/12372532/174259974-5926009a-5525-47d5-9b02-d65dd64ddd12.png)

**Comment by souredoutlook** [comment]
Hi @Rainnut 

It's hard to investigate this without knowing more about your setup - are you using self-hosted Sentry or is the SaaS product? If it's SaaS I can have a close look if you share the org and project slugs.
